### DISCUSSION TITLE
Task 2: Money Change

### DISCUSSION THREAD

**Discussion by rojo32** [rramosp/ai4eng.v1#17] [02 - PYTHON]
Saludos

Profe me puede dar una  pista  para la solución del ejercicio, he realizado el proceso de varias maneras y no me da el valor esperado

**Comment by rramosp** [comment]
hola rojo32 ... deberías de formular una pregunta concreta. Con un caso de uso concreto (inputs) y una pregunta específica sobre tu código

Te recomiendo (1) probar con casos sencillos; y (2) entender (print) qué valores intermedios generas en tu solución.

**Reply by rojo32** [reply]
gracias por su respuesta profe,

Tengo un error en el codigo ya que no me da los valores esperado en la posicion r[0]

1. 
![image](https://user-images.githubusercontent.com/5897347/120054778-a0c4a000-bff7-11eb-9361-44b6a27b9347.png)

**Reply by rramosp** [reply]
si lo haces paso a paso, con lápiz y papel, deberías de darte cuenta de cual es el error al calcular r[0]
